test/test.wpcom.site.taxonomy.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Module dependencies
+ */
+var util = require( './util' );
+var assert = require( 'assert' );
+
+/**
+ * site.follow
+ */
+describe( 'wpcom.site.taxonomy', function() {
+	// Global instances
+	var wpcom = util.wpcom();
+	var site = wpcom.site( util.site() );
+	var taxonomy = site.taxonomy( 'category' );
+
+	describe( 'wpcom.site.taxonomy.termsList', function( done ) {
+		it( 'should return a list of terms', done => {
+			taxonomy.termsList()
```

Comment:
Instead of accepting calling `done`, Mocha can handle promises which should work just as well here I'd imagine:

```js
it( 'should return a list of terms', () => {
	return taxonomy.termsList().then( ( data ) => {
		// ...
	} );
} );
```